doc/manual/manual.tex

```diff
@@ -12484,7 +12484,7 @@ \subsubsection{Running tests}
 compiled \aspect{}.
 
 After running \texttt{cmake} and then compiling \aspect{}, you can run the
-testsuite by saying \texttt{ctest}. By default, this will only run a small
+testsuite by using the command \texttt{ctest} in your build directory. By default, this will only run a small
 subset of all tests given that both setting up all tests (several hundred) and
 running them takes a non-trivial amount of time. To set up the full test suite,
 you can run
```

Comment:
Missing text after "you can run" ?

Reply:
There is text after "you can run", it's just a command:

```
you can run
\begin{lstlisting}[frame=single,language=ksh]
    make setup_tests
\end{lstlisting}
in the build directory. 
```
I don't know why these lines are not shown on github, maybe the manual.tex file is too long.

Reply:
Weird. If that text is present on your local copy of manual.tex, it must indeed be a github issue?

Reply:
Makes sense. I just gave this a second read through and it looks good to go. I'm not sure why the jenkins tester is not running, but for this case it does not really matter. Unless you would like another set of eyes on the changes, feel to merge when ready.

Reply:
The jenkins tester is not running because someone has to start it and I don't have the permission to do that... but as you pointed out, all changes are in the manual.tex, so it can't break the tester anyway.

Reply:
And I believe if you think it's ready to merge, you should just be brave and hit the merge bottun :grinning:

Reply:
merged 😄